crates/harness-tests/src/routing.rs

```diff
@@ -199,3 +200,29 @@ async fn monolith_double_load_room(ctx: &mut TestRunner) {
     let t = resp.text().await.expect("failed to read http response");
     assert_eq!(t, "{\"name\":\"foo\"}");
 }
+
+#[test_context(TestRunner)]
+#[tokio::test]
+async fn unicast_messaging(ctx: &mut TestRunner) {
+    let mut m = Monolith::new(ctx).await.unwrap();
+
+    m.show().await;
+
+    let mut c1 = Client::new(ctx).unwrap();
+    c1.join("foo").await;
+    let mut c2 = Client::new(ctx).unwrap();
+    c2.join("foo").await;
+
+    m.wait_recv().await;
+
+    let c_id = m.clients().iter().next().copied();
+
+    m.send(M2BRoomMsg {
+        room: "foo".into(),
+        client_id: c_id,
+        payload: serde_json::json!({}),
```

Comment:
This will fix the compile error
```suggestion
        payload: RawValue::from_string("{}".to_owned()).unwrap(),
```